function/examples/README.md

```diff
@@ -1,8 +1,37 @@
-# Examples 
+# function-examples
 
 ## Overview
 
-The following examples show how to use Function API to:
+This module contains examples showing how to use Function API.
 
-* [Initialize a workspace](./init/README.md)
-* [Apply a Function to a cluster](./apply/README.md)
+## Operators
+
+Operators allow to perform two operations on given resources: `apply` and `delete`. 
+Both operations may accept callbacks that will be executed before and/or after each operation.
+
+The library contains two implementations of operator interface: `GenericOperator` and `TriggersOperator`.
+
+Examples:
+
+* [apply operation](./cmd/operator/apply/main.go)
+* [delete operation](./cmd/operator/delete/main.go)
+
+## Callbacks
+
+Callbacks are optional functions provided by function API user to be executed before and/or after each operation. User may chain callbacks by providing multiple functions.
```

Comment:
```suggestion
Callbacks are optional Functions that you can provide to execute them before and/or after each operation. You can chain callbacks by providing multiple Functions.
```

Reply:
`may` -> `can` is a preferential change but it is simply more "tangible" this way. I also changed passive voice to active as this is the preferable form in our docs.